coq-hierarchy-builder-shim.opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "2.0"
+name: "coq-hierarchy-builder"
+version: "dev"
+maintainer: "Enrico Tassi <enrico.tassi@inria.fr>"
+authors: [ "Cyril Cohen" "Kazuhiko Sakaguchi" "Enrico Tassi" ]
+license: "MIT"
+homepage: "https://github.com/math-comp/hierarchy-builder"
+bug-reports: "https://github.com/math-comp/hierarchy-builder/issues"
+dev-repo: "git+https://github.com/math-comp/hierarchy-builder"
+
+build: [ make "-C" "shim" "build" ]
+install: [ make "-C" "shim" "install" ]
+conflicts: [ "coq-hierarchy-builder" ]
```

Comment:
Looks to me we'd like to have the same for nix...

Reply:
I guess so but I don't know nix well enough to know how...